components/base-node.js

```diff
@@ -44,6 +46,18 @@ module.exports = {
    },
 
   /**
+   * Standard component error handling - send a message to the error port and throw Error
```

Comment:
FYI, the 'assign' function above (line 17/19) will go away when we get a uniform wrapper API implemented, as I mentioned in email.